index.ts

```diff
@@ -168,17 +168,7 @@ commander
     localhost: boolean;
     ownkey: boolean;
   }) => {
-    // Try to read the RC file.
-    DOTFILE.RC.read().then((rc: ClaspSettings) => {
-      console.warn(ERROR.LOGGED_IN);
-    }).catch(async (err: string) => {
-      DOTFILE.RC_LOCAL.read().then((rc: ClaspSettings) => {
-        console.warn(ERROR.LOGGED_IN);
-      }).catch(async (err: string) => {
-        await checkIfOnline();
-        authorize(options.localhost, options.ownkey);
-      });
-    });
+    login(options);
```

Comment:
If you look at this, we're calling a function with `options` that just calls a function with `options`.

I bet you can just do this:
```ts
commander
  .command('login')
  .description('Log in to script.google.com')
  .option('--no-localhost', 'Do not run a local server, manually enter code instead')
  .option('--ownkey', 'Save .clasprc.json file to current working directory')
  .action(login);
```